Pipfile

```diff
@@ -18,4 +18,4 @@ django-rest-api = "*"
 [dev-packages]
 
```

Comment:
I would have added in a code formatter like black or autopep8. It would help keep your formatting clean and consistent.